assets/js/app.js

```diff
@@ -42,23 +42,16 @@ _.padLeft = function(s, l, c) {
 };
 
 var nav = $('nav.global');
-$(window).scroll(_.throttle(function(){
-  nav.toggleClass('full', $(window).scrollTop() > 0);
-}, 100));
+
+nav.addClass('full');
+  $('.full-only').addClass('active');
```

Comment:
In `assets/styles/layout/_nav.scss`, (125-126), `.full-only` will have its `display` property set as `block` if `nav` has the `full` class.

Why do we need to add the `active` class to `full-only`?